tools/hmac/payload2.json

```diff
@@ -0,0 +1,61 @@
+{
+    "data": {
+      "balancePlatform": "YOUR_BALANCE_PLATFORM",
```

Comment:
Rename payload2.json so that it's clear it's a platform webhook .json file

Reply:
again just an example, also it can be used for Management API webhooks